sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/scm/ScmAccountService.java

```diff
@@ -77,6 +92,29 @@ public ScmAccount getScmAccountByRepo(String repo) {
         return scmAccountMapper.getAllScmAccounts();
     }
 
+    @Transactional(readOnly = true)
+    public List<Organization> getScmAccountOrganizations(Long id) throws IOException {
+        ScmAccount scmAccount = getScmAccountById(id);
+        return gitHubService.getOrganizations(scmAccount);
+    }
+
+    @Transactional(readOnly = true)
+    public List<Repository> getScmAccountRepositories(Long id, String organizationName) throws IOException {
+        ScmAccount scmAccount = getScmAccountById(id);
+        List<ScmAccount> allAccounts = getAllScmAccounts();
+        List<String> repositoryUrls = allAccounts.stream()
+                                                 .map(ScmAccount::getRepositoryURL)
+                                                 .collect(Collectors.toList());
+        List<Repository> repositories = gitHubService.getRepositories(scmAccount, organizationName);
```

Comment:
Maybe we should filter those at gitHubService as well by passing org name + repo url? And maybe even filter those at DB level by passing those to query